scripts/vendor

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
CI seems to be missing this file

Reply:
It won’t once the parent PR is merged in shipyard.

Reply:
Ah, okay.

Reply:
This script is called by scripts/entry hence the CI fails regardless of shipyard's state.

Now should or should it not be called by scripts/entry is a different question :)
My guess is since you now have ` $(TARGETS): vendor/modules.txt` then it should be fine as it's handled by make, so you can remove that call from the entry script

Reply:
Ah yes, thanks. In fact I should add the `vendor` handling to submariner and share the `entry` script...

Reply:
I guess you mean add it to shipyard as an "sh" script in the shared folder..? :)